kvmagent/kvmagent/plugins/host_model_mount_plugin.py

```
@@ -0,0 +1,189 @@
+        rsp.mountPoint = mount_point
+
+        # Mount
+        success, error = mount_juicefs(cmd.zdfsUrl, mount_point)
+        if success:
+            rsp.success = True
+            logger.info("Successfully mounted Model Center[%s] at %s" % (
+                cmd.modelCenterUuid, mount_point))
+        else:
+            rsp.error = error
+
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)